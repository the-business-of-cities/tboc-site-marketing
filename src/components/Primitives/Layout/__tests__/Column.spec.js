import { Column } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Column", () => {
  it("renders correctly", () => {
    expect(shallow(<Column />)).toMatchSnapshot();
  });
});
