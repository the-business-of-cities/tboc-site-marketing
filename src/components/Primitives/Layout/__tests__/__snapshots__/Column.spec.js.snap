// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders correctly 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c0:not(:last-child) {
  padding-left: 0.5em;
}

.c0:not(:first-child) {
  padding-right: 0.5em;
}

@media (max-width:999px) {
  .c0 {
    -webkit-flex-basis: 100% !important;
    -ms-flex-preferred-size: 100% !important;
    flex-basis: 100% !important;
    padding: 1em;
  }
}

@media (min-width:768px) {
  .c0:first-child {
    -webkit-flex: 1;
    -ms-flex: 1;
    flex: 1;
  }

  .c0:last-child {
    -webkit-flex: 1;
    -ms-flex: 1;
    flex: 1;
  }

  .c0 > img,
  .c0 > a > img {
    padding: 2em;
  }
}

<div
  className="c0"
/>
`;
